nft-example/tests/sample.rs

```diff
@@ -0,0 +1,284 @@
+use codec::Encode;
+use gtest::{Program, System};
+use nft_example_io::*;
+const USERS: &'static [u64] = &[3, 4, 5];
+
+fn init_with_mint(sys: &System) {
+    sys.init_logger();
+
+    let nft = Program::from_file(
+        &sys,
+        "../../apps/target/wasm32-unknown-unknown/release/nft_example.wasm",
+    );
+
+    let res = nft.send(
+        USERS[0],
+        InitConfig {
+            name: String::from("MyToken"),
+            symbol: String::from("MTK"),
+            base_uri: String::from(""),
+        },
+    );
+
+    assert!(res.log().is_empty());
```

Comment:
What happens if initialization fails?

Reply:
It shouldn't fail here since that initialization is correct